test/ng/locationSpec.js

```diff
@@ -1601,6 +1601,25 @@ describe('$location', function() {
     });
 
 
+    it('should rewrite links when the specified rewriteLinks attr is detected', function() {
+      configureTestLink({
+        linkHref: 'link?a#b',
+        attrs: 'force-rewrite',
+        html5Mode: {enabled: true, rewriteLinks:false},
+        supportHist: true
+      });
+      initService({html5Mode:{enabled: true, rewriteLinks:'forceRewrite'},supportHistory:true});
```

Comment:
Unless I am wrong, this test should fail (due to https://github.com/angular/angular.js/pull/14976/files#r73798407), but there are currently no failures on Travis (at least not one related to this test :wink:). Which makes me wonder, does this test even run? Needs investigation...